addons/mail/static/src/discuss/core/common/message_actions.js

```diff
@@ -8,15 +8,15 @@ import { rpc } from "@web/core/network/rpc";
 registerMessageAction("set-new-message-separator", {
     condition: ({ message, thread }) =>
         thread &&
-        thread.self_member_id &&
+        thread.channel?.self_member_id &&
```

Comment:
Isn't this action 100% linked to channel? Shouldn't we use `action.channel` then?